packages/turf-geojson-rbush/tsconfig.testTypes.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "compilerOptions": {
+    "noEmit": true
+  },
+  "files": ["types.ts"]
+}
```

Comment:
If you wanted to consolidate this further, you can make a root `tsconfig.testTypes.json` which has all of this and then each package can just be an `extends` directive. With `${configDir}` things like `files` are now possible to define once. https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-configdir-template-variable-for-configuration-files

Reply:
That would be great! Wasn't able to find a way to avoid these until now. I will refactor.

Reply:
https://github.com/Turfjs/turf/pull/2684 is the Typescript upgrade for Turf

Reply:
Shall we get #2684 merged first then, I'll bring that into this PR, and then rejig to use the configDir stuff?

Reply:
Yeah feel free to merge it whenever you want, I hit the update button. Actually though this brings up a bit of an interesting point, a new typescript version (or relevant package.json change?) should force everything to rebuild/test.

Reply:
If you mean in CI, then yes it did.